portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,135 @@
 <!DOCTYPE html>
-<html>
+<html lang="en-US" dir="ltr">
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="viewport" content="wisth=device-width, initial-scale=1">
+
+    <title>Arrington's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <header>
+          <h1>arrington.</h1>
+          <!-- maybe add a leaderboard ad - they're 728 x 90 -->
+          <nav>
+              <a>home</a>
+              <a>about</a>
+              <a>projects</a>
+              <a>contact</a>
+              <a>skills</a>
+              </ul>
+          </nav>
+      </header>
+    
+
+    <!-- elements to add:
+        contacting arrington
+        arrington's interests (done)
+        arrington's details/skills (done)
+        arrington's schools (done)
+        arrington's companies (done)
+        arrington's blurbs
+        arrington's friend space
+        favorite song
+        projects + relevant coursework
+        my header or whatever it's called (done)
+         -->
+
+    <section id="about" class="left">
+        <h2>arrington</h2>
+        <!-- insert image -->
+        <p>"welcome to my portfolio :-)"</p>
+        <p>she/her</p>
+        <p>18 years old</p>
+        <p>Birmingham, ALABAMA United States</p>
+        <p>Last login: 08/01/2003</p>
```

Comment:
Maybe in the future this date can be populated from the server

Reply:
Ooh, good idea! I added a TODO for that in the javascript file so I don't forget.